src/MongoDB.Driver.Core/Core/Operations/RetryableDeleteCommandOperation.cs

```diff
@@ -75,16 +77,6 @@ public BatchableSource<DeleteRequest> Deletes
             get { return _deletes; }
         }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether the server should process the deletes in order.
-        /// </summary>
-        /// <value>A value indicating whether the server should process the deletes in order.</value>
-        public bool IsOrdered
```

Comment:
The IsOrdered property was duplicated in three classes. It's been moved to the base class.